src/components/transactions/Warnings/MarketWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import { Trans } from '@lingui/macro';
+import { Link, Typography, AlertColor } from '@mui/material';
+
+import { Warning } from '../../primitives/Warning';
+
+interface MarketWarningProps {
+  learnMore?: boolean; // Modify wording on link text,
+  warningMessage: string;
+  linkHref?: string;
+  warningType: AlertColor;
+}
+
+export const MarketWarning = ({
+  learnMore,
+  warningMessage,
+  linkHref,
+  warningType,
+}: MarketWarningProps) => {
+  return (
+    <Warning severity={warningType}>
+      <Typography variant="caption">
+        <Trans>
+          {warningMessage}{' '}
```

Comment:
I think these props may need to be passed in with the <Trans> tag at the higher level.  Looking into this a couple months ago exposed that issue, where Crowdin can't work through the strings coming in as props and then being translated, since they could be dynamic.

Reply:
It's not a very scalable system, but with this translation tag, I haven't found another way yet that dynamic translations can happen. I don't think it's even possible though... and the props are handled at runtime.

Reply:
Yep yep, noticed right as I pushed and fixed with latest commit

Also removed some of the unnecessary props here